recipe/meta.yaml

```diff
@@ -12,23 +12,26 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: './setup.sh && cp -r $SRC_DIR $PREFIX'
```

Comment:
@synapticarbors, `cp -r $SRC_DIR $PREFIX` is not working as previously because the name of SRC_DIR changed.
You need to do something like,
```
./setup.sh && mkdir -p $PREFIX/$PKG_NAME-$PKG_VERSION && cp -r $SRC_DIR/* $PREFIX/$PKG_NAME-$PKG_VERSION
```

Reply:
Using the `script:  ...` command suggested in your comment seems to result in the same error. In your suggestion that the post-link script isn't needed, are you basically proposing that the `script` section in the recipe yaml and the `post-link.sh` should be replaced with a `build.sh` that combines the two?

Reply:
yes